api/test_resources/boxes.json

```diff
@@ -0,0 +1,1341 @@
+      },
+      "witness": {
+        "goType": "generated.ApplicationResponse",
+        "app": {
+          "application": {
+            "created-at-round": 1,
+            "deleted": false,
+            "id": 3,
+            "params": {
+              "approval-program": "CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=",
+              "clear-state-program": "CIEB",
+              "creator": "N5T74SANUWLHI6ZWYFQBEB6J2VXBTYUYZNWQB2V26DCF4ARKC7GDUW3IRU",
+              "global-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              },
+              "local-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              }
+            }
+          },
+          "current-round": 8
+        }
+      },
+      "witnessError": null
+    },
+    {
+      "name": "Lookup app (funded with boxes)",
+      "request": {
+        "path": "/v2/applications/1",
+        "params": [],
+        "url": "http://localhost:8999/v2/applications/1",
+        "route": ""
+      },
+      "response": {
+        "statusCode": 200,
+        "body": "{\"application\":{\"created-at-round\":1,\"deleted\":false,\"id\":1,\"params\":{\"approval-program\":\"CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=\",\"clear-state-program\":\"CIEB\",\"creator\":\"GJR76Q6OXNZ2CYIVCFCDTJRBAAR6TYEJJENEII3G2U3JH546SPBQA62IFY\",\"global-state-schema\":{\"num-byte-slice\":0,\"num-uint\":0},\"local-state-schema\":{\"num-byte-slice\":0,\"num-uint\":0}}},\"current-round\":8}\n"
+      },
+      "witness": {
+        "goType": "generated.ApplicationResponse",
+        "app": {
+          "application": {
+            "created-at-round": 1,
+            "deleted": false,
+            "id": 1,
+            "params": {
+              "approval-program": "CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=",
+              "clear-state-program": "CIEB",
+              "creator": "GJR76Q6OXNZ2CYIVCFCDTJRBAAR6TYEJJENEII3G2U3JH546SPBQA62IFY",
+              "global-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              },
+              "local-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              }
+            }
+          },
+          "current-round": 8
+        }
+      },
+      "witnessError": null
+    },
```

Comment:
Notice that this app-info response _does not_ contain any information about boxes. To find the summary information for a app's boxes, instead, one will need to query the app's _app-account_ (see below for an example).

@winder @jannotti